<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Floor Matt</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Feet</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Inches</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>sixteenths</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Section Length</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QSpinBox" name="SectLenInchesSpinBox"/>
    </item>
    <item row="1" column="4">
     <widget class="QSpinBox" name="SectLenSixteenthsSpinBox"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Section Width</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QSpinBox" name="SectionWidthInchesSpinBox"/>
    </item>
    <item row="2" column="4">
     <widget class="QSpinBox" name="SectionWidthSixteenthstSpinBox"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Board Width</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QSpinBox" name="BoardWidthInchesSpinBox"/>
    </item>
    <item row="3" column="4">
     <widget class="QSpinBox" name="BoardWidthSixteenthsSpinBox"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Board Length</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QSpinBox" name="BoardLengthInchesSpinBox"/>
    </item>
    <item row="4" column="4">
     <widget class="QSpinBox" name="BoardLengthSixteenthsSpinBox"/>
    </item>
    <item row="5" column="0" colspan="5">
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Generate Report</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QSpinBox" name="SectLenFeetSpinBox"/>
    </item>
    <item row="2" column="2">
     <widget class="QSpinBox" name="SectionWidthFeetSpinBox"/>
    </item>
    <item row="3" column="2">
     <widget class="QSpinBox" name="BoardWidthFeetSpinBox"/>
    </item>
    <item row="4" column="2">
     <widget class="QSpinBox" name="BoardLengthFeetSpinBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>448</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
